docs/src/arch-introduction.md

```
@@ -1,35 +1,32 @@
-You would need CruiseKube if you are facing any of these issues -
-
-- **Chronic over-provisioning** driven by guesswork, peak-based sizing, and fear of CPU throttling or OOM crashes
-- **Cost inefficiency** that node-level bin packing as provided by autoscalers (Cluster Autoscaler/Karpenter) alone cannot fix
-- **Operational Load** arising from manual tuning of workloads on kubernetes by developers or DevOps teams
-
-CruiseKube explicitly addresses the **pod-level right-sizing problem**, in a fully hands-off manner.
-
-## Architecture
+Together, these tasks allow CruiseKube to continuously optimize resources **without relying on manual tuning or reactive scaling**.
 
+## Components
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)